src/server/_query.py

```diff
@@ -115,25 +115,25 @@ def filter_fields(generator: Iterable[Dict[str, Any]]):
             yield filtered
 
 
-def filter_geo_pairs(
+def filter_geo_sets(
     type_field: str,
     value_field: str,
-    values: Sequence[GeoPair],
+    values: Sequence[GeoSet],
     param_key: str,
     params: Dict[str, Any],
 ) -> str:
     """
-    returns the SQL sub query to filter by the given geo pairs
+    returns the SQL sub query to filter by the given geo sets
     """
 
-    def filter_pair(pair: GeoPair, i) -> str:
+    def filter_set(set: GeoSet, i) -> str:
```

Comment:
similar to other comment about builtin; perhaps use `gset` or `g_set` instead of just `set` here?